derrida/books/models.py

```diff
@@ -677,8 +406,6 @@ class ReferenceType(Named, Notable):
 class Reference(models.Model):
     '''Reference to a book from a work by Derrida.  Can be a citation,
     quotation, or other kind of reference.'''
-    book = models.ForeignKey(Book)
```

Comment:
I think this was the change I was worried about that I thought I might have missed, but I looked again and the books to works data migration includes a line to update references.

Although instance should probably be required now that book is removed...